tasks/src/visual-question-answering/about.md

```diff
@@ -0,0 +1,43 @@
+## Use Cases
+
+### Automatic Medical Diagnostic
+Visual Question Answering (VQA) models can be used to help clinical staff diagnose conditions. For example, a doctor could input an MRI scan to the model and ask "what does this MRI scan show?" and get the answer to their question. 
```

Comment:
We discussed with @meg-huggingface about this (given she studied ethical side of VQA task) it's better if we don't include this (the model can pick up spurious cues and might answer questions wrongly in life critical situations)